### DISCUSSION TITLE
GitHub falsely suspended my account.

### DISCUSSION THREAD

**Discussion by asamolion** [community/community#23324] [New to GitHub]
<p>I was simply working from my office in Pakistan when suddenly I started seeing a warning on my GitHub.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/89158a0b17c37ef570d72cce80db10b86e346a70.png" data-download-href="/uploads/short-url/jyHyMEc4mvfuHco7qJorY7POILu.png?dl=1" title="Screenshot from 2020-03-19 20-18-04.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/14261053/181076138-5d99c802-eb8c-4e2f-b08e-9936116ad74c.png" alt="Screenshot from 2020-03-19 20-18-04.png" data-base62-sha1="jyHyMEc4mvfuHco7qJorY7POILu" width="690" height="103" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/89158a0b17c37ef570d72cce80db10b86e346a70_2_690x103.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/89158a0b17c37ef570d72cce80db10b86e346a70_2_1035x154.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/89158a0b17c37ef570d72cce80db10b86e346a70_2_1380x206.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/89158a0b17c37ef570d72cce80db10b86e346a70_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2020-03-19 20-18-04.png</span><span class="informations">1866×280 58.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>After that I’m not able to access any of my private repos or the repos of the organization I’m working for (<a href="http://Jumpcut.com" rel="nofollow noopener">Jumpcut.com</a>)</p>
<p>I’ve filed an appeal with proof of my residence but according to <a href="https://github.community/t/github-support-ignores-my-requests-for-the-third-week/2543">this thread</a> it may be weeks before I’m able to access it again.</p>
<p>I would really appreciate if someone from GitHub would get this matter resolved. I am unable to work till then using this account.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Our sincere apologies that it took so long to respond to your report here.</p>
<p>In working with the support team that reviews these items, it appears that your account was unflagged in the intervening time and things should be working normally for you.</p>
<p>If you’re running into any other issues, please let us know!</p>
